doc/init.rst

```diff
@@ -0,0 +1,11 @@
+:orphan:
+
+Create dummy project
+--------------------
+
+After installing Universum via ``pip3.7 install -U universum`` or simply from a downloaded project root directory
+run ``python3.7 -m universum init`` to create an example source directory with example config and get a command
+to run it.
+
+After doing so replace ``universum_dummy_project`` with actual project sources, and edit ``universum_config.py``
```

Comment:
Is comma missing here?

Reply:
I'd just replace "After doing so" with "Afterwards"